### DISCUSSION TITLE
steam版即将发售，赠送序列号🎉

### DISCUSSION THREAD

**Discussion by hanaTsuk1** [shion-app/shion#37] [Announcements]
> 原来还有steam版吗?

在闲谈中，曾经聊到过[付费模式](https://shion.app/zh/chat/payment-model)

感谢开源社区的支持，将于7月1日20点赠送steam序列号🥳

**Comment by hanaTsuk1** [comment]
占个位置😶‍🌫️

分发结束

**Comment by q943613777** [comment]
在哪送

**Reply by hanaTsuk1** [reply]
就在这

**Reply by hanaTsuk1** [reply]
3C2LR-2KJCD-MJLFX

**Reply by q943613777** [reply]
> 3C2LR-2KJCD-MJLFX

激活了，感谢

**Comment by zhao-wuyan** [comment]
赠送数量有多少啊

**Reply by hanaTsuk1** [reply]
BG47M-75PV6-T2965
别问有多少  我怕等会一车面包人🤣

**Reply by zhao-wuyan** [reply]
哈哈好，谢谢作者大大。

**Comment by megumiss** [comment]
还有么(●'◡'●)

**Reply by hanaTsuk1** [reply]
AQRAD-ZH5J2-F53T9

**Reply by megumiss** [reply]
感谢

**Comment by xu-Fencer** [comment]
大佬大佬，我也想要

**Reply by hanaTsuk1** [reply]
5ZKMR-H72PA-I9K3R

**Reply by xu-Fencer** [reply]
感谢，顺便想问下如何把原来版本的数据迁移到steam版本中，`C:\Users\***\AppData\Local`目录中有`shion` `hanaTsuk1.shion` `hanaTsuk1.shion.pro`三个文件，夹把旧的复制到新的是否能成功迁移数据，或者有没有其他更好的办法

**Reply by hanaTsuk1** [reply]
有没有一种可能，视图下面就有导入与导出

[指南](https://shion.app/zh/guide/general#%E5%A6%82%E4%BD%95%E8%BF%9B%E8%A1%8C%E6%95%B0%E6%8D%AE%E8%BF%81%E7%A7%BB)里也有

**Reply by xu-Fencer** [reply]
> 有没有一种可能，视图下面就有导入与导出
> 
> [指南](https://shion.app/zh/guide/general#%E5%A6%82%E4%BD%95%E8%BF%9B%E8%A1%8C%E6%95%B0%E6%8D%AE%E8%BF%81%E7%A7%BB)里也有

好的，最近打开软件只看总览把这些功能给忘了，已经成功迁移

**Comment by WinHex89** [comment]
不好意思，来晚两天，刚看到有这么好的软件，求开发者赏一个，我本人有运营一个公众呈，打算给这个软件宣传宣传。

**Reply by hanaTsuk1** [reply]
08AVB-9DMPY-DGM36
有一点要说一下，不要把这个帖子发出去

**Reply by WinHex89** [reply]
感谢感谢，不会发的，刚想去steam下载，结果发现自己steam账号被盗了，得申诉回来后再下载软件了😅

**Comment by Celosia-argentea** [comment]
已在Steam支持！真的是一个很棒的软件！

**Reply by hanaTsuk1** [reply]
非常感谢🥳

**Reply by hanaTsuk1** [reply]
8AMIH-IZACG-BGNAI
送给你身边需要的人吧

**Reply by Celosia-argentea** [reply]
感谢您！

**Comment by wcgjob** [comment]
大佬，给我一个吧

**Reply by hanaTsuk1** [reply]
C362F-M0NY8-2XBMX

**Reply by wcgjob** [reply]
谢谢大佬

**Comment by Manicok** [comment]
还有吗？大佬以后能不能再做一个计划、标签关联文件的功能，比如在写 .md 就自动开始计时笔记计划；打开某个pdf文件就计时读某本书的标签 。

**Reply by hanaTsuk1** [reply]
XRLQP-AAK4Y-6LZ6K
下个版本的功能我还在考虑，你说的这个我知道了，实不实现以后再说

**Comment by kelomina** [comment]
大佬给个

**Reply by hanaTsuk1** [reply]
G7IW3-YHWIY-ZKGBC

**Comment by guiyyo** [comment]
大佬，还有机会吗
